xarray/backends/netCDF4_.py

```diff
@@ -115,13 +118,6 @@ def _nc4_group(ds, group, mode):
         return ds
 
 
-def _ensure_fill_value_valid(data, attributes):
```

Comment:
I pushed it back but initially removed it because it wasn't called but have put it back.